products.css

```diff
@@ -0,0 +1,53 @@
+
+
+#fruits {
+    padding: 0;
+    display: flex;
+    flex-wrap: wrap;
+}
+
+#fruits li {
```

Comment:
two things: the # sign is for IDs the . is for classes.  You'd never have a bunch of items with the same ID.  Also, it doesn't seem as though your fruit items even have the fruit class.  so if you just delete the #fruit part off of each of your css blocks and target the li directly....it looks fantastic!